matrix_io/malos/driver.py

```diff
@@ -0,0 +1,175 @@
+"""
+MALOS -- Hardware abstraction layer for MATRIX Creator usable via 0MQ
+
+This is an asyncio based implementation of the MALOS drivers. It uses
+coroutines to handle communications with 0MQ and provides a higher
+level API for client code.
+
+You can find more details about the MALOS protocol here:
+https://github.com/matrix-io/matrix-creator-malos
+
+List of base ports
+------------------
+
+IMU with port 20013.
+Humidity with port 20017.
+Everloop with port 20021.
+Pressure with port 20025.
+UV with port 20029.
+MicArray_Alsa with port 20037.
+FACE with port 60001
+
+Each port reserves a range of 4 ports that are used for a driver:
+
+* Base port: (driver port) port for sending configuration proto to MALOS driver.
+* Keep alive port: port for sending keep alive messages, so MALOS driver keeps reading data
+  from sensors.
+* Error port: port yielding MALOS errors when they occur.
+* Data update port: port yielding sensor data updates.
+
+"""
+
+import asyncio
+
+import zmq
+from zmq.asyncio import Context
+
+IMU_PORT = 20013
+HUMIDITY_PORT = 20017
+EVERLOOP_PORT = 20021
+PRESSURE_PORT = 20025
+UV_PORT = 20029
+MICARRAY_ALSA_PORT = 20037
+FACE_PORT = 60001
+
+
+class MalosDriver(object):
+
+    def __init__(self, address, base_port, config_proto):
```

Comment:
Perhaps we could remove `config_proto`  from here ? Maybe it's ok to init without a config so it can be passed later ?